src/components/Search/SearchResultsListItem.tsx

```diff
@@ -66,7 +72,19 @@ function SearchResultsListItem({
   const ListItemComponent = listItemComponent === 'radio' ? Radio : Checkbox;
   const revisionHash = truncateHash(item.revision);
   const commitMessage = getLatestCommitMessage(item);
-  const itemDate = new Date(item.push_timestamp * 1000);
+
+  const itemDate = dayjs.unix(item.push_timestamp);
+
+  const date = itemDate.format('MM/DD/YY');
```

Comment:
The screenshot Carla shared in https://github.com/mozilla/perfcompare/issues/767 shows the date and time using the USA notation: MM/DD/YY. But this is confusing to all other users. Indeed in most other parts of the world, we use DD/MM/YY. Therefore for some dates it's confusing. For example 01/10/24 could be either January 10 or October 1st.

Therefore my recommendation would be to use `MMM` instead of `MM`, so that the first letters of the month is used. Also it would be better to use a single digit for the day when possible, so use `D` instead of `DD`.
Then I believe the format would look like `MMM D, YY`, but please check if this looks right.

This wasn't mentioned in the issue, sorry about that.